src/components/FoodOrderComponents/OrderPlace.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+    <v-container class="orderPlaceStyle">
+        <!-- Header & Input -->
+        <v-row justify="center" no-gutters>
+            <h2>מאיפה תרצו להזמין?</h2>
+        </v-row>
+
+        <v-row justify="center" no-gutters>
+            <v-col class="placeTextFieldStyle" cols="3">
+                <v-text-field
+                        :append-outer-icon="placeInput ? 'navigate_before' : ''"
+                        @click:append-outer="sendPlaceInput"
+                        @click:clear="clearPlaceInput"
+                        clear-icon="mdi-close-circle"
+                        clearable
+                        filled
+                        label="מקום"
+                        type="text"
+                        v-model="placeInput"
+                ></v-text-field>
+            </v-col>
+        </v-row>
+
+        <!-- Places list -->
+        <v-row justify="center" no-gutters>
+            <v-col cols="3" class="placeListsCol">
+                <v-progress-circular :size="40" color="primary"
+                                     indeterminate v-if="$apollo.loading"/>
+                <v-container class="overflow-y-auto placesContainer" v-if="!$apollo.loading">
+                    <v-list shaped="true" v-scroll>
+                        <v-list-item-group color="primary" v-model="chosenPlaceIndex">
+                            <v-list-item :key="place.id" v-for="place in places">
+                                <v-list-item-content>
+                                    <v-list-item-title v-html="place.name"></v-list-item-title>
+                                </v-list-item-content>
+                            </v-list-item>
+                        </v-list-item-group>
+                    </v-list>
+                </v-container>
+            </v-col>
+        </v-row>
+
+        <!-- Footer & Buttons -->
+        <v-row justify-md="end" no-gutters
+               v-if="chosenPlaceIndex !== undefined && chosenPlaceIndex !== -1">
```

Comment:
How can this be undefined?

Reply:
Double selection of the same element will make chosenPlaceIndex to be undefined.